.github/workflows/deploy-ecs.yml

```diff
@@ -0,0 +1,23 @@
+name: Deploy to ECS
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        required: true
+        default: dev
+  push:
+    branches: ["master"]
```

Comment:
This will automatically run a deploy to ECS whenever a branch is merged to master. We want that for dev, but not prod. I've seen other teams split this up into separate workflows so they can control the triggers for each deploy type (see Skate's [set-up](https://github.com/mbta/skate/tree/master/.github/workflows)). (I don't think there's a way to do that with one workflow, and I'm not too worried with trying to force it into one workflow.)

Reply:
Actually, nevermind! I'm learning about the idea of "inputs" here. I'll slack you.